packages/backend/helpers/rateLimitMiddleware.ts

```diff
@@ -0,0 +1,54 @@
+import { Request, Response } from 'express';
+import { RateLimiterRedis, IRateLimiterStoreOptions } from 'rate-limiter-flexible';
+
+import redis from '../redis/client';
+import { skipRateLimitRoutes } from './utils';
+
+//We can make this dynamic based on the subscription as well
+const RATE_LIMIT_DURATION_IN_MINUTES = 1;
+//TODO:Should we keep DEFAULT_RATE_LIMIT_DEVELOPER_PLAN value in env variable?
```

Comment:
@jatinsandilya inputs

Reply:
@shadab14meb346 Lets put `DEFAULT_RATE_LIMIT_DEVELOPER_PLAN` in the `.env` file. Do remember to the update the `.env.example` as well for any future changes.